frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx

```diff
@@ -27,57 +29,88 @@ const fetchSubmissionsFromUser = async (submission_id: string) => {
     });
 };
 
-const columns: GridColDef<Submission>[] = [
-  { field: "submission_id", headerName: "Submission ID", flex: 0.4 },
-  { field: "display_name", headerName: "Student", width: 160, flex: 0.4 },
-  {
-    field: "grading",
-    headerName: "Grading",
-    editable: true,
-    flex: 0.2,
-  },
-  {
-    field: "submission_status",
-    headerName: "Status",
-    renderCell: (params: GridRenderCellParams<Submission>) => (
-      <>
-        {params.row.submission_status === "SUCCESS" ? (
-          <CheckCircleIcon sx={{ color: green[500] }} />
-        ) : (
-          <CancelIcon sx={{ color: red[500] }} />
-        )}
-      </>
-    ),
-  },
-  {
-    field: "submission_path",
-    headerName: "Download",
-    renderCell: (params: GridRenderCellParams<Submission>) => (
-      <IconButton
-        onClick={() => fetchSubmissionsFromUser(params.row.submission_id)}
-      >
-        <DownloadIcon />
-      </IconButton>
-    ),
-  },
-];
+const editGrade = (submission: Submission, oldSubmission: Submission, errorMessage: string) => {
+  const submission_id = submission.submission_id;
+  const newGrade = submission.grading;
+
+  if (newGrade < 0 || newGrade > 20) {
+    alert(errorMessage);
```

Comment:
No using alert

Reply:
What do i need to use then?

Reply:
https://github.com/SELab-2/UGent-3/pull/355/commits/f74ea45daaf697b134a0b9ccc2390f81d0e529dd